docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx

```diff
@@ -640,7 +640,75 @@ selfservice:
 Next, open the login endpoint of the SecureApp and you should see the Facebook
 Login option!
 
-## Google, LinkedIn
+## Google
+
+**Set up on Google Cloud Platform**
+
+1. Create an
+   [OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent)
+2. Create
+   [OAuth 2.0 Client IDs](https://console.cloud.google.com/apis/credentials)
+
+In the new _OAuth 2.0 Client ID_, you will need the `Client ID` and
+`Client secret`.
+
+**Configuring Kratos**
+
+Create a Jsonnet claims mapper as described in
+[OpenID Connect and OAuth2 Credentials](../concepts/credentials/openid-connect-oidc-oauth2.mdx).
+Save the code in
+`<kratos-directory>/contrib/quickstart/kratos/email-password/oidc.google.jsonnet`.
+
+```json title="contrib/quickstart/kratos/email-password/oidc.google.jsonnet"
+local claims = {
+  email_verified: true
+} + std.extVar('claims');
+
+{
+  identity: {
+    traits: {
+      [if "email" in claims && claims.email_verified then "email" else null]: claims.email,
+      first_name: claims.given_name,
+      last_name: claims.family_name,
+      [if "hd" in claims && claims.email_verified then "hd" else null]: claims.hd,
```

Comment:
Can you explain here what the hd claim is doing?

Reply:
Sorry, I should clarify - there should be a section on g suite and the HD claim under the `## Google` tag, otherwise it is not clear what this claim is and why we use it or need it here.

Reply:
Will add more information to the docs :)